pkgs/development/python-modules/pass2csv/default.nix

```diff
@@ -0,0 +1,22 @@
+{ buildPythonApplication, fetchFromGitHub, python-gnupg, lib}:
+
+buildPythonApplication rec {
```

Comment:
Applications don't belong in `python-modules`, please move to all-packages

More information on this can be found [here](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#topythonapplication-function)

Reply:
The commit msg should then be:
```
pass2csv: init at 0.3.2
```

Reply:
my bad i forgot to update the pr title